app/controllers/notes_controller.rb

```diff
@@ -55,10 +55,56 @@ def destroy
       track_destroyed(@note)
       redirect_to node_path(@node), notice: 'Note deleted'
     else
-      redirect_to node_note_path(@node, @note), alert: 'Could not delete node'
+      redirect_to node_note_path(@node, @note), alert: 'Could not delete note'
     end
   end
 
+  def multi_destroy
+    @notes = Note.where(
+      'node_id = :node AND id in (:ids)',
+      node: @node,
+      ids: params[:ids]
+    )
+
+    if @notes.any?
```

Comment:
This triggers it's own SQL request.

Since further down (L72) you're using `#count`, why don't you capture the count outside of the `if/end` and then use `if count > 0` as a condition.

I believe it will save us one SELECT